public/video-ui/styles/main.scss

```diff
@@ -19,6 +19,7 @@
 
 // 4. Layout-related sections
 @import
+  'layout/_common.scss',
```

Comment:
Is this different to having the name missing the underscore (as below)?

Reply:
yes, in that this is wrong 😛